save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -52,12 +54,12 @@ class Contest(
     fun toDto() = ContestDto(
         name,
         status,
-        startTime!!,
-        endTime!!,
+        startTime?.toKotlinLocalDateTime()!!,
```

Comment:
I think the previous version, when we just had a typealias with expect/actual implementation looked better...

Reply:
yeap, it will be great to have alias in kotlinx.serialization instead of a dedicated class